packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -120,6 +120,11 @@ const propTypes = {
    * If the `variant` prop if set to `12-hour` for one of these supported locales, the variant will be ignored and defaults to `24-hour`.
    */
   timeVariant: PropTypes.oneOf([DateTimeUtils.FORMAT_12_HOUR, DateTimeUtils.FORMAT_24_HOUR]),
+  /**
+   * Timezone value to indicate in which timezone the date-time component is rendered.
+   * The value provided should be a valid [timezone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) string, else will default to browser/local timezone.
+   */
+  timeZone: PropTypes.string,
```

Comment:
As this is a string. are we validating the timezone prop value ?

Reply:
Yes, we are validating. 

Its done [here](https://github.com/cerner/terra-framework/pull/1044/files/b14377f4f34f42d937a36f052d1386c43c50eeb3#diff-0a8a568512ba2dc166f3b71e47749752R260).

`moment.tz.zone(timeZone)` validates the timeZone string.